mlrun/runtimes/mpijob/abstract.py

```diff
@@ -296,3 +353,48 @@ def with_autotune(
             ] = gaussian_process_noise
 
         self.set_envs(horovod_autotune_settings)
+
+    def set_mpi_args(self, args: typing.List[str]) -> None:
+        """Sets the runtime's mpi arguments to args.
+
+        Parameters
+        ----------
+        args : typing.List[str]
+            Arguments to be used for the mpi-operator
+
+        Raises
+        ------
+        ValueError
+            args is of type `List[str]` and can only accept `str` parameters.
+
+        Example
+        -------
+        ```
+        # Define the wanted MPI arguments
+        mpi_args = []
+        mpi_args.append('-x')
+        mpi_args.append('NCCL_DEBUG=INFO')
+        mpi_args.append('-x')
+        mpi_args.append('NCCL_SOCKET_NTHREADS=2')
+        mpi_args.append('-x')
+        mpi_args.append('NCCL_NSOCKS_PERTHREAD=8')
+        mpi_args.append('-x')
+        mpi_args.append('NCCL_MIN_NCHANNELS=4')
+
+        # Set the MPI arguments in the function
+        fn.set_mpi_args(mpi_args)
```

Comment:
I don't see a reason for so many lines, why not just:
```
mpi_args = ['-x', 'NCCL_DEBUG=INFO', '-x', 'NCCL_SOCKET_NTHREADS=2', '-x', 'NCCL_NSOCKS_PERTHREAD=8', '-x', 'NCCL_MIN_NCHANNELS=4']

# Set the MPI arguments in the function
fn.set_mpi_args(mpi_args)
```

Reply:
Since this is intended for the docs anyway, it seems easier to read and for the users who copy to edit.
if you insist i'll change